src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int DEFAULT_RADIUS = 10;
+    public static final String DEFAULT_COLOR = "WHITE";
+    public static final int FIGURE_BOUND = 5;
+    public static final int RANDOM_NUMBER = 10;

```

Comment:
strange naming for a constant don't you think? :)
```suggestion
    public static final int RANDOM_NUMBER_BOUND = 10;

```